.circleci/config.yml

```diff
@@ -54,21 +57,84 @@ jobs:
           name: Build and push Docker image
           command: |
             echo $DOCKER_PASSWORD | docker login -u $DOCKER_LOGIN --password-stdin
+            docker-compose -f production.yml pull
             docker-compose -f production.yml build
             docker-compose -f production.yml push
+
+  build_frontend:
+    docker:
+      - image: circleci/node:11.15
+    steps:
+      - checkout
       - run:
-          name: Deploy
+          name: Create .env file
           command: |
-            ssh -o StrictHostKeyChecking=no $HOST_USERNAME@$HOSTNAME "/bin/bash ./deploy.sh"
+            echo -e "REACT_APP_API_BASE=$REACT_APP_API_BASE\nREACT_APP_GOOGLE_CLIENT_ID=$REACT_APP_GOOGLE_CLIENT_ID" > frontend/.env.production
+      - restore_cache:
+          key: dependency-cache-{{ checksum "frontend/package.json" }}
+      - run:
+          name: Install dependencies
+          command: "cd frontend && npm install"
+      - save_cache:
+          key: dependency-cache-{{ checksum "frontend/package.json" }}
+          paths:
+            - ./frontend/node_modules
+      - run:
+          name: Build React frontend
+          command: "cd frontend && npm run build"
+      - persist_to_workspace:
+          root: frontend/build
+          paths:
+            - .
+
+  deploy_frontend:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - attach_workspace:
+          at: build
+      - aws-s3/sync:
+          from: build/
+          to: "s3://$AWS_S3_BUCKET/frontend"
+          overwrite: true
+      - run:
+          name: Invalidate CloudFront cache
+          command: |
+            aws cloudfront create-invalidation --distribution-id $AWS_CLOUDFRONT_DISTRIBUTION_ID --paths '/*'
+
+  deploy_backend:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - run:
+          name: Deploy backend
+          command: |
+            ssh -o StrictHostKeyChecking=no $HOST_USERNAME@$HOSTNAME "/bin/bash /sprints/deploy.sh"
```

Comment:
Can you disable passing credentials to forks on the CI configuration? 
This is to ensure that our production environment will be safe, and no variables will be leaked through the CI.

Reply:
@giovannicimolin, it is [disabled by default](https://circleci.com/docs/2.0/oss/#security). You can check that it's disabled [here](https://circleci.com/gh/open-craft/sprints/edit#advanced-settings).